src/main/java/seedu/address/commons/core/UserSettings.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.commons.core;
```

Comment:
Nice, simple design. But why didn't you use the UserPrefs class in the model package?